src/components/MoneyRequestHeader.js

```diff
@@ -153,7 +153,7 @@ const MoneyRequestHeader = (props) => {
                                     iouReport={props.report}
                                     onPress={(paymentType) => IOU.payMoneyRequest(paymentType, props.chatReport, props.report)}
                                     enablePaymentsRoute={ROUTES.BANK_ACCOUNT_NEW}
-                                    addBankAccountRoute={ROUTES.IOU_DETAILS_ADD_BANK_ACCOUNT}
+                                    addBankAccountRoute={ROUTES.getBankAccountRoute('', props.report.policyID)}
```

Comment:
We should only pass the policyID if this is an Expense report. For IOUs we don't need the policyID since it'd be the user personal bank account and not the Workspace bank account. In the case of expense report, the policyID on the chat and expense reports are the same.